definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js

```diff
@@ -0,0 +1,128 @@
+// @flow
+
+declare module 'express' {
+    declare type RouterOptions = {
+      caseSensitive?: boolean,
+      mergeParams?: boolean,
+      strict?: boolean
+    };
+    declare class RequestResponseBase {
+        app: Application;
+        get(field: string): string | void;
+    }
+    declare class Request extends http$IncomingMessage mixins RequestResponseBase {
+      baseUrl: string;
+      body: any;
+      cookies: { [cookie: string]: string },
+      fresh: boolean;
+      hostname: boolean;
+      ip: string;
+      ips: Array<string>,
+      method: string,
+      originalUrl: string;
+      params: { [param: string]: string },
+      path: string,
+      protocol: 'https' | 'http',
+      query: { [name: string]: string },
+      route: string,
+      secure: boolean,
+      signedCookies: { [signedCookie: string]: string },
+      stale: boolean;
+      subdomains: Array<string>;
+      xhr: boolean;
+      accepts(types: string): string | false;
+      acceptsCharsets(...charsets: Array<string>): string | false;
+      acceptsEncodings(...encoding: Array<string>): string | false;
+      acceptsLanguages(...lang: Array<string>): string | false;
+      header(field: string): string | void;
+      is(type: string): boolean;
+      param(name: string, defaultValue?: string): string | void;
+    }
+
+
+    declare type CookieOptions = {
+        domain?: string,
+        encode?: (value: string) => string,
+        expires?: Date,
+        httpOnly?: boolean,
+        maxAge?: string,
+        path?: string,
+        secure?: boolean,
+        signed?: boolean
+    };
+
+    declare type SendFileOptions = {
+        maxAge?: number,
+        root?: string,
+        lastModified?: boolean,
+        headers?: { [name: string]: string },
+        dotfiles?: 'allow' | 'deny' | 'ignore'
+    };
+    declare class Response extends http$ClientRequest mixins RequestResponseBase {
+        headersSent: boolean;
+        locals: any;
+        append(field: string, value?: string): void;
+        attachment(filename?: string): void;
+        cookie(name: string, value: string, options?: CookieOptions): void;
+        clearCookie(name: string, options?: CookieOptions): void;
+        download(path: string, filename?: string, callback?: (err?: ?Error) => void): void;
+        format(typesObject: {[type: string]: Function}): void;
+        json(body?: any): void;
```

Comment:
Let's [switch these params to `mixed`](https://github.com/flowtype/flow-typed/blob/master/CONTRIBUTING.md#avoid-any-when-possible)